controlled_access_terms.tokens.inc

```diff
@@ -0,0 +1,364 @@
+<?php
+
+/**
+ * @file
+ * Contains controlled_access_terms.tokens.inc.
+ *
+ * This file provides islandora tokens.
+ */
+
+use Drupal\taxonomy\Entity\Term;
+use Drupal\media\Entity\Media;
+use Drupal\file\Entity\File;
+use Drupal\controlled_access_terms\EDTFUtils;
+use Drupal\Core\Render\BubbleableMetadata;
+
+/**
+ * Implements hook_token_info().
+ */
+function controlled_access_terms_token_info() {
+  $type = [
+    'name' => t('Islandora Tokens'),
+    'description' => t('Tokens for Islandora objects.'),
+  ];
+  $node['title'] = [
+    'name' => t("Node Title"),
+    'description' => t("The node's title"),
+  ];
+  $node['agent_author'] = [
+    'name' => t("Author"),
+    'description' => t("The node's Author linked agents"),
+  ];
+  $node['agent_contributor'] = [
+    'name' => t("Contributors"),
+    'description' => t("The node's Contributor linked agents"),
+  ];
+  $node['agent_publisher'] = [
+    'name' => t("Publishers"),
+    'description' => t("The node's Publishers linked agents"),
+  ];
+  $node['agent_creators'] = [
+    'name' => t("Creators"),
+    'description' => t("The node's Creators linked agents"),
+  ];
+  $node['publication_date'] = [
+    'name' => t("Publication date"),
+    'description' => t('Show the "Date Created" into YYYY/MM/DD format (handles EDTF format)'),
+  ];
+  $node['pdf_url'] = [
+    'name' => t("PDF Url"),
+    'description' => t('URL to related media file if "Original file" is a PDF file'),
+  ];
+  $node['media-thumbnail-image:url'] = [
```

Comment:
`media-thumbnail-image:url` and `media-thumbnail-alt` as well as their underscored counterparts can be removed now that they're in the islandora module.